tests/tests/swfs/avm2/stage3d_raytrace/test.toml

```diff
@@ -0,0 +1,16 @@
+# Note - this test runs the 'RayTracing' example
+# from the https://github.com/saharan/OGSL repository
+# All of the necessary files were copied into this directory.
+
+num_frames = 80
+
+[image_comparison]
+tolerance = 0
+# FIXME - see if we can decrease this. Ideally, it wouldn't be nearly so high
+max_outliers = 5100
+
+[player_options]
+# This test runs very slowly on unoptimized `test` builds,
+# so give it plenty of time.
+max_execution_duration = { secs = 1000, nanos = 0 }
+with_renderer = { optional = false, sample_count = 1 }
```

Comment:
I think renderer would be required for this, no?

Reply:
That's right - this is `optional = false`, so it's required.

Reply:
Oh gosh sorry, misread it!